### PR TITLE
completed assignment

### PR DESCRIPTION
**Pull request by drewfrench**
~

### CODE REVIEW

File: assignment2.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Drew French
+
+2)	a)	false || (true && false)
+		false || false
+		false
```
Comment (oldclesleycode) [inline assignment2.txt:5]: Right!

File: assignment2.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+	
+	b)	true && true && true
+		true && true
+		true
```
Comment (oldclesleycode) [inline assignment2.txt:9]: Yup! 👍

File: assignment2.txt [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+	
+	c)	false || true
+		true
```
Comment (oldclesleycode) [inline assignment2.txt:12]: right!

File: assignment2.txt [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+3)	It will give a runtime error; there should be &userIn
```
Comment (oldclesleycode) [inline assignment2.txt:14]: Correct!

File: assignment2.txt [hunk 5]
```diff
@@ -0,0 +1,15 @@
+Drew French
+
```
Comment (oldclesleycode) [inline assignment2.txt:2]: You're missing an answer to a question.

File: evenodd.c [hunk 6]
```diff
@@ -0,0 +1,25 @@
+/* Drew French */
+
+/* Determines if a number input
```
Comment (oldclesleycode) [inline evenodd.c:3]: Very nice job! 💯

File: threenums.c [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+			d = a;
+		}
+		if(b > a && b > c)
+		{
+			d = b;
+		}
+		if(c > a && c > b)
+		{
+			d = c;
+		}
```
Comment (oldclesleycode) [inline threenums.c:41]: Nice job, but because of the way your logic is implemented, your code doesn't properly handle cases where two of the numbers are equal and also the largest number. Make sure to test for edge cases.
